sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -258,6 +258,64 @@ public virtual async Task<Response<KeyVaultKey>> CreateRsaKeyAsync(CreateRsaKeyO
             }
         }
 
+        /// <summary>
+        /// Creates and stores a new AES key in Key Vault. If the named key already exists, Azure Key Vault creates a new
+        /// version of the key. It requires the keys/create permission.
```

Comment:
nit for consistency: 
```suggestion
        /// version of the key. This operation requires the keys/create permission.
```